dashboard/lib/policies/level_files.rb

```diff
@@ -32,8 +32,8 @@ def self.level_name_from_path(path)
 
     # Return a glob which can be used either to find the specified level file
     # by name if provided, or all level files if not.
-    def self.level_file_glob(level_name)
-      level_name ? "config/levels/**/#{level_name}.level" : 'config/levels/**/*.level'
+    def self.level_file_glob(level_name, root_dir = '.')
```

Comment:
My goal with these was to leave the behavior unchanged for any other callers that currently don't pass parameters. I assumed that relative paths would be resolved relative to the current working directory rather than `Rails.root`. Was that assumption mistaken?

Reply:
Good point, I don't know the answer but it looks like you're making a best-effort attempt to preserve existing behavior, and that seems like the right level of effort here. if we want import_concept_difficulty_tags.rb to keep working over time then we should add test coverage next time we need to use it.